Actions/WorkflowInitialize/WorkflowInitialize.ps1

```diff
@@ -9,18 +9,27 @@ try {
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-TestRepoHelper.ps1" -Resolve)
 
-    $ap = "$ENV:GITHUB_ACTION_PATH".Split('\')
+    $ap = "$ENV:GITHUB_ACTION_PATH".Split([System.IO.Path]::DirectorySeparatorChar)
```

Comment:
Is this related to the AL-Go deployment bug? 🤔 If not, it probably belongs in a different PR

Reply:
I will move the workflowinitialize bug fixes to a different PR.